### PR TITLE
Fixed IE issue

### PR DESCRIPTION
**Pull request by linkfang**
fixes [issue171](https://github.com/EclipseFdn/react-eclipsefdn-members/issues/171)

### CODE REVIEW

File: src/main/www/public/index.html [hunk 1]
```diff
@@ -90,6 +90,7 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
     <!-- Placed at the end of the document so the pages load faster -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.6.15/browser-polyfill.min.js"></script>
```
Comment (autumnfound) [inline src/main/www/public/index.html:93]: If we are going to reference a lib like this we should take a copy during the webpack and reference that instead. This way if we go to new versions, or that page goes away, we still have an up to date local copy.

Comment (linkfang) [reply src/main/www/public/index.html]: Ah, OK.
I thought that was the same one we were using in index.tsx, but it is not.

So, I updated the solution. As long as the polyfill is loaded before react and react-dom, then IE won't have any issue about it.

Comment (autumnfound) [reply src/main/www/public/index.html]: Can you confirm that this works in the preview for IE?

Comment (linkfang) [reply src/main/www/public/index.html]: Yep, it works fine with the new solution

**Review by autumnfound — APPROVED** [review]

**Pull request merged**